build.cake

```diff
@@ -111,10 +111,12 @@ Task("PublishCalamariProjects")
                      DotNetCorePublish(project.FullPath, new DotNetCorePublishSettings
 		    	    {
 		    	    	Configuration = configuration,
-                        OutputDirectory = $"{publishDir}/{calamariFlavour}/{platform}",
+                        OutputDirectory = $"{publishDir}{calamariFlavour}/{platform}",
                         Framework = framework,
                         Runtime = runtime
 		    	    });
+                    Console.WriteLine($"{publishDir}{calamariFlavour}/{platform}");
```

Comment:
Create a Calamari zip per runtime to makes it easier for Teamcity build configs to grab the right version of Calamari